docker/ssl/Caddyfile

```diff
@@ -1,7 +1,7 @@
 # https://caddyserver.com/docs/caddyfile
 {
 	#debug
-	#auto_https disable_redirects
+	auto_https disable_redirects
```

Comment:
this will disabled the service worker locally... I'd like to think through this a little more together before we move forward.

Reply:
does it really disable the service worker?

I would expect the service worker to fail if the application is accessed through HTTP, which is what this change allows.  If the application is accessed via HTTPS e.g. https://localhost I would expect the service worker to work normally.

This change simply disables auto_redirect allowing the dev to choose whether they want http or https.  That's my read of what this setting does in any case.

Reply:
yes, you're right, I'm getting things mixed up, this is probably ok.  We should just review the docs and make sure we're calling out the idea of being able to use both http and https and when to use each.

Reply:
I added a [note that clarifies that both HTTP and HTTPS are available](https://github.com/sillsdev/web-languageforge/pull/1479/commits/cc01515e06e4b19c96236f6978de27ae079ecdef), but HTTPS is required for service-workers.

Reply:
thank you!